src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
```

Comment:
you've add it, that's OK now, but you don't use it 😄
```suggestion
  constructor(name) {
    this.name = name;
```